src/OneExplorer.ts

```diff
@@ -172,29 +195,122 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
+  // TODO(dayo) extract file-relative functions as another module
+  private parseInputPath = (configPath: string, modelPath: string): string|undefined => {
+    const config = readIni(configPath);
+    const ext = path.extname(modelPath).slice(1);
+
+    if (ext === undefined || config === null) {
+      return undefined;
+    }
+
+    return config[`one-import-${ext}` as keyof typeof config]?.['input_path'];
+  };
+
+  // TODO(dayo) extract file-relative functions as another module
+  private grepTargetInCommand = (str: string): string[] => {
+    return str.split(' ').filter(e => path.extname(e) === '.tvn' || path.extname(e) === '.circle');
+  };
+
+  // TODO(dayo) extract file-relative functions as another module
+  private grepAll = (str: string): string[] => {
+    return [str];
+  };
+
+  // TODO(dayo) extract file-relative functions as another module
+  private parseIntermediates = (configPath: string): string[] => {
+    const config = readIni(configPath);
```

Comment:
Maybe is same configfile read more than one time at L200 and then L222?
If so, maybe this could be fixed later.

Reply:
I may pass parsed init as argument to the function, but I didn't see the needs as basically our ini file is small that it won't affect performance.